ci/platforms/config.hera

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/bash
 
-export GFS_CI_ROOT=/scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT
+export GFS_CI_ROOT=/scratch1/NCEPDEV/global/glopara/GFS_CI_CD
 export ICSDIR_ROOT=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
 
+# Jenkins directories
+export JENKINS_AGENT_LANUCH_DIR=${GFS_CI_ROOT}/Jenkins/agent
+export JENKINS_WORK_DIR=${GFS_CI_ROOT}/Jenkins/workspace
+# JENKINS cutome_workspace directory where CI jobs are run
+# /scratch1/NCEPDEV/global/glopara/CI
+
+# CTest functional test directories for pre stagged input data
 export STAGED_TESTS_DIR=${GFS_CI_ROOT}/STAGED_TESTS_DIR
+
+# CI BASH test directories
+export GFS_BASH_CI_ROOT=${GFS_CI_ROOT}/Bash
```

Comment:
```suggestion
export GFS_BASH_CI_ROOT=${GFS_CI_ROOT}/GFS_BASH_CI
```
To be consistent w/ other platform configs.

Reply:
Yes lets follow form.